src/base/scanfoldersmodel.cpp

```diff
@@ -153,10 +153,10 @@ QVariant ScanFoldersModel::headerData(int section, Qt::Orientation orientation,
 
     switch (section) {
     case WATCH:
-        title = tr("Watched Folder");
+        title = tr("Monitored Folder");
         break;
     case DOWNLOAD:
-        title = tr("Save Files to");
+        title = tr("Overwrite Save Location");
```

Comment:
>The correct word is "override". 

Ah yes, corrected.

>But why not simply "Save Location" ?

The "Override" word is eye-catching (probably...), so users will notice that there is a option.